CHANGELOG.txt

```diff
@@ -1,3 +1,212 @@
+- Fixes wrong limit for dask for python>3.7 (should be <3.7) (#15545)
+- Fix Task Adoption in ``KubernetesExecutor`` (#14795)
+- Fix timeout when using XCom with KubernetesPodOperator (#15388)
+- Fix deprecated provider aliases in "extras" not working (#15465)
+- Fixed default XCom deserialization. (#14827)
+- Fix used_group_ids in ``dag.partial_subset`` (#13700) (#15308)
+- Further fix trimmed `pod_id` for `KubernetesPodOperator` (#15445)
+- Bugfix: Invalid name when trimmed `pod_id` ends with hyphen in ``KubernetesPodOperator`` (#15443)
+- Fix incorrect slots stats when TI ``pool_slots > 1`` (#15426)
+- Fix DAG last run link (#15327)
+- fix broken link in experimental API deprecation headers (#13547)
+- Fix typo in task_runner for ``AirflowConfigException`` (#15067)
+- Fix ``sync-perm`` to work correctly when update_fab_perms = False (#14847)
+- Fixes limits on Arrow for plexus test (#14781)
+- Fix UI bugs in tree view (#14566)
+- Fix AzureDataFactoryHook failing to instantiate its connection (#14565)
+- Fix permission error on non-POSIX filesystem (#13121)
+- Fix spelling in "ignorable" (#14348)
+- Fix get_context_data doctest import (#14288)
+- Correct typo in GCSObjectsWtihPrefixExistenceSensor  (#14179)
+- Fix order of failed deps (#14036)
+- Fix critical CeleryKubernetesExecutor bug (#13247)
+- Fix four bugs in StackdriverTaskHandler (#13784)
+- ``func.sum`` may return ``Decimal`` that break rest APIs (#15585)
+- Persist tags params in pagination (#15411)
+- API: Raise `AlreadyExists` exception when the execution_date is same (#15174)
+- Remove duplicate call to sync_metadata inside DagFileProcessorManager (#15121)
+- extra docker-py update to resolve docker op issues (#15731)
+- Ensure executors end method is called (#14085)
+- Remove user_id from API schema (#15117)
+- Removes references to directory that no longer exists (#15083)
+- Prevent clickable bad links on disabled pagination (#15074)
+- Acquire lock on db for the time of migration (#10151)
+- Skip SLA check only if SLA is None (#14064)
+- Print right version in airflow info command (#14560)
+- Make ``airflow info`` work with pipes (#14528)
+- Rework client-side script for connection form. (#14052)
+- Add CollectionInfo in all Collections that have total_entries (#14366)
+
+Doc only changes
+""""""""""""""""
+
+- Fix docstring of SqlSensor (#15466)
+- Small changes on "DAGs and Tasks documentation" (#14853)
+- Add note on changes to configuration options (#15696)
+- Add docs to the markdownlint and yamllint config files (#15682)
+- Rename old "Experimental" API to deprecated in the docs. (#15653)
+- Fix documentation error in `git_sync_template.yaml` (#13197)
+- Fix doc link permission name (#14972)
+- Fix link to Helm chart docs (#14652)
+- Fix docstrings for Kubernetes code (#14605)
+- docs: Capitalise & minor fixes (#14283) (#14534)
+- Fixed reading from zip package to default to text. (#13984)
+- Fix spellings (#13867)
```

Comment:
can be removed -- generally I tend to removed "spelling/typo" ones -- ones that are not CHANGELOG-worthy